doc/input-output-reference/src/overview/group-zone-forced-air-units.tex

```diff
@@ -1484,9 +1484,9 @@ \subsubsection{Inputs}\label{inputs-4-040}
+  \textbf{ZoneTemperatureDeadbandOnOffCycling}. This control method determines whether or not to operate the cooler based on the thermostat setpoint and the zone air temperature.~ This thermostatic control method uses a throttling temperature range, determined in the following input field, to model hysteresis-type control to avoid excessive short-cycling.~ If the zone air temperature is warmer than the cooling setpoint temperature plus one-half of the throttling range, then the unit is operated for the entire timestep at the design air mass flow rate.~ If the zone air temperature is cooler than the cooling setpoint temperature minus one-half of the throttling range, then the unit is off for the entire timestep.~ When the zone air temperatures are within the throttling range, then the unit will stay off if it was not running during the previous timestep and will stay on if it was already running. If the unit is On, then the unit is cycled such that it meets the current cooling load and avoids over-cooling the zone.
 \item
-  \textbf{ZoneCoolingLoadOnOffCycling}. This control method determines whether or not to operate the cooler based on the predicted zone load to cooling setpoint.~ If there is a cooling load (and the unit is available), then the unit is operated for the entire timestep at the design air mass flow rate. If there is no cooling load, then the unit is not operated at all.~ The magnitude of the cooling load that will trigger the unit to operate can be controlled using the field below called Cooling Load Control Threshold Heat Transfer Rate. This control method does not usually behave in a smooth manner and zone conditions will tend to oscillate considerably during periods of low load.
+  \textbf{ZoneCoolingLoadOnOffCycling}. This control method determines whether or not to operate the cooler based on the predicted zone load to cooling setpoint.~ If there is a cooling load (and the unit is available), then the unit is operated for the entire timestep at the design air mass flow rate. If there is no cooling load, then the unit is not operated at all.~ The magnitude of the cooling load that will trigger the unit to operate can be controlled using the field below called Cooling Load Control Threshold Heat Transfer Rate. If the unit is On, then the unit is cycled such that it meets the current cooling load and avoids over-cooling the zone.
```

Comment:
For this control type, seems it should read "If there is a cooling load (and the unit is available), then the unit is cycled at the design air mass flow rate such that it meets the current cooling load . . . " (essentially merging the last sentence with the second sentence).

Reply:
Updated as suggested.